core/config/project_settings.cpp

```diff
@@ -55,11 +55,7 @@ String ProjectSettings::get_resource_path() const {
 const String ProjectSettings::IMPORTED_FILES_PATH("res://.godot/imported");
 
 String ProjectSettings::localize_path(const String &p_path) const {
-	if (resource_path == "") {
-		return p_path; //not initialized yet
-	}
-
-	if (p_path.begins_with("res://") || p_path.begins_with("user://") ||
+	if (resource_path == "" || p_path.begins_with("res://") || p_path.begins_with("user://") ||
```

Comment:
[Cosmetic] Wouldn't it be better to change the first condition `resource_path == ""` into `resource_path.is_empty`? 
```
if (resource_path.is_empty() || p_path.begins_with("res://") || p_path.begins_with("user://") ||
```

Reply:
Yes, it's also more efficient, thanks for the tip :+1: